Platform_FVP_Corstone_SSE-300_Ethos-U55/run_example.cmd

```diff
@@ -1,2 +1,2 @@
 @echo off
-"C:\Program Files\ARM\VHT\models\Win64_VC2019\VHT-Corstone-300.exe" -V "..\VSI\audio\python" -f fvp_config.txt -a Objects\microspeech.axf --stat --cyclelimit 2400000000 %*
+"C:\Program Files\ARM\VHT\models\Win64_VC2019\VHT_Corstone_SSE-300_Ethos-U55" -V "..\VSI\audio\python" -f fvp_config.txt -a Objects\microspeech.axf --stat --cyclelimit 2400000000 %*
```

Comment:
We should keep the .exe extension.
The renamed executable also needs to be reflected in the uVision project - I will take care of that.